services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -457,4 +513,20 @@ class SwitchRulesSpec extends BaseSpecification {
 
         return rules
     }
+
+    void checkTrafficCountersInRules(FlowEndpointPayload flowEndpoint, trafficThroughIngressRule = true) {
```

Comment:
This is personal, but I find that the `trafficThroughIngressRule` should not be optional, since method signature without this param looks unintuitive